src/Bridges/lazy_bridge_optimizer.jl

```diff
@@ -152,7 +152,12 @@ Return the `Edge` or `ObjectiveEdge` in the hyper-graph associated with the
 bridge `BT`, where `index` is the index of `BT` in the list of bridges.
 """
 function _edge(b::LazyBridgeOptimizer, index::Int, BT::Type{<:AbstractBridge})
-    return Edge(index, _variable_nodes(b, BT), _constraint_nodes(b, BT))
+    return Edge(
+        index,
+        _variable_nodes(b, BT),
+        _constraint_nodes(b, BT),
+        bridging_cost(BT),
```

Comment:
One question is whether this should be `bridging_cost(b, BT)`. Will we ever need to query something in the model?

Reply:
I thought the same. The current approach is to keep the bridging API simple and avoid `added_constraint_types`, `supports_constraint`, ... take the model as argument for to force the bridges to be simple and easier to reason about. The dependence with the model is solely handled by the bridged optimizer and the bridge graph